### ISSUE TITLE
Metformin

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#155]
https://en.wikipedia.org/wiki/Metformin

**Comment by nelsonic** [comment]
Doing a bit of light evening reading in the **Journal of Advanced Experimental Medical Biology** ...  

The Use of Metformin to Increase the Human Healthspan:
https://pubmed.ncbi.nlm.nih.gov/32304040/

Metformin is a safe, effective and useful drug for glucose management in patients with diabetes. However in recent years, more attention has been paid to the possibility of using metformin as an anti-aging drug. It was shown to significantly increase the lifespan in some model organisms and delay the onset of age-associated declines. The current review summarizes advances in clinical research on the potential role of metformin in the field of lifespan and healthspan extension. Growing amounts of evidence from clinical trials suggest that metformin can effectively reduce the risk of many age-related diseases and conditions, including cardiometabolic disorders, neurodegeneration, chronic inflammation and frailty. Metformin also holds promise as a drug that could be repurposed for chemoprevention or adjuvant therapy for certain types of cancer. Moreover, metformin induces autophagy by activation of AMPK and can thus be potentially used to promote heathspan by hormesis-like mechanisms. Although long-term intake of metformin is associated with low risk of adverse events, well-designed clinical trials are still required to uncover the potential use of this drug as a geroprotector.

Metformin - its potential anti-cancer and anti-aging effects:
https://pubmed.ncbi.nlm.nih.gov/28258677/

The effects of metformin on autophagy: https://pubmed.ncbi.nlm.nih.gov/33524789/

Metformin treatment alleviates polycystic ovary syndrome: https://pubmed.ncbi.nlm.nih.gov/30989649/

Clinical trials and observational studies uncovered that metformin can remarkably prevent or alleviate cardiovascular diseases, obesity, polycystic ovarian syndrome (PCOS), osteoporosis, cancer, periodontitis, neuronal damage and neurodegenerative diseases, inflammation, inflammatory bowel disease (IBD), tuberculosis, and COVID-19. In addition, metformin has been proposed as an antiaging agent. Numerous mechanisms were shown to be involved in the protective effects of metformin. Metformin activates the LKB1/AMPK pathway to interact with several intracellular signaling pathways and molecular mechanisms. The drug modifies the biologic function of NF-κB, PI3K/AKT/mTOR, SIRT1/PGC-1α, NLRP3, ERK, P38 MAPK, Wnt/β-catenin, Nrf2, JNK, and other major molecules in the intracellular signaling network. It also regulates the expression of noncoding RNAs. Thereby, metformin can regulate metabolism, growth, proliferation, inflammation, tumorigenesis, and senescence. Additionally, metformin modulates immune response, autophagy, mitophagy, endoplasmic reticulum (ER) stress, and apoptosis and exerts epigenetic effects. Furthermore, metformin protects against oxidative stress and genomic instability, preserves telomere length, and prevents stem cell exhaustion. In this review, the protective effects of metformin on each disease will be discussed using the results of recent meta-analyses, clinical trials, and observational studies. Thereafter, it will be meticulously explained how metformin reprograms intracellular signaling pathways and alters molecular and cellular interactions to modify the clinical presentations of several diseases.

**Comment by iteles** [comment]
Did you find anything on the body's learned resistance to it?

**Comment by nelsonic** [comment]
@iteles yes, around **10%** of Diabetics became intolerant of the lower doses (i.e. they had to increase their dose from 500mg to 850mg. 

Metformin efficacy and tolerance in obese non-insulin dependent diabetics:
https://pubmed.ncbi.nlm.nih.gov/2702849/

> Thankfully I am not diabetic or even pre-diabetic because I've systematically **eliminated refined sugar** from my diet over the last 2 years (except for cake/icecream on very special occasions e.g. birthdays) so my resting blood glucose level is exactly at the "healthy" level; not too low/high.

I'm investigating Metformin from the perspective of its' _other_ benefits.
The fact that it lowers blood glucose level means it protects your heart, liver, kidneys and brain.

However if you look it up on NutritionFacts.org, the best way to prevent _all_ cause mortality is diet & exercise.
https://nutritionfacts.org/?s=metformin

**Comment by nelsonic** [comment]
Metformin first appeared on our radar in 2019 when we listened to **Lifespan** on car journeys: https://lifespanbook.com/ 
David Sinclair https://www.linkedin.com/in/sinclairda/ is a Professor at Harvard Medical School studies longevity and age-related diseases

https://youtu.be/bRWT7hVgwuM?t=3207


![image](https://user-images.githubusercontent.com/194400/159577871-007f343a-08dc-4c28-83b3-3db5afb37441.png)

![image](https://user-images.githubusercontent.com/194400/159578052-7a78541a-2e08-4b2b-93b9-9ff0d7c82e9a.png)


**Side note**:  watching the video in Chrome this was the Ad I was subjected to while watching a video about longevity: 
![image](https://user-images.githubusercontent.com/194400/159577776-2ade9cab-fca0-44e1-b177-ecc485e22830.png)

Advertising is _not_ your friend. 😞 


https://lifespanbook.com/talks/
![image](https://user-images.githubusercontent.com/194400/159579853-ca882e22-a545-46de-b5e2-8b561d7d57bc.png)

Podcast notes: https://podcastnotes.org/lifespan-with-dr-david-sinclair/episode-4-nmn-nr-resveratrol-metformin-other-longevity-molecules-the-science-of-setting-achieving-goals-lifespan-with-dr-david-sinclair/

https://podcastnotes.org/podcast-notes-summaries/david-sinclairs-supplement-regimen-nmn-resveratrol-metformin-vitamin-d-vitamin-k-and-the-science-of-longevity/

Dr. David Sinclair takes Metformin: **1 g** at Night.

**Comment by nelsonic** [comment]
Metformin: historical overview: https://pubmed.ncbi.nlm.nih.gov/28776081/

The ability of metformin to counter insulin resistance and address adult-onset hyperglycaemia without weight gain or increased risk of hypoglycaemia gradually gathered credence in Europe, and after intensive scrutiny metformin was introduced into the USA in 1995. Long-term cardiovascular benefits of metformin were identified by the UK Prospective Diabetes Study (UKPDS) in 1998, providing a new rationale to adopt metformin as initial therapy to manage hyperglycaemia in type 2 diabetes. Sixty years after its introduction in diabetes treatment, metformin has become the most prescribed glucose-lowering medicine worldwide with the potential for further therapeutic applications.

**Comment by nelsonic** [comment]
1 in 10,000 probability of Lactic acidosis: https://en.wikipedia.org/wiki/Metformin#Adverse_effects

**Comment by nelsonic** [comment]
Metformin: Current knowledge: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4214027/

Lactic acidosis usually occurs due to drug overdose or in some contraindicated conditions. It is more likely to occur in patients with certain medical conditions, including a serious infection, liver or kidney disease, recent surgery, any conditions that cause a low level of oxygen in the blood or poor circulation (such as recent stroke, congestive heart failure, recent heart attack), heavy alcohol use, dehydration, X-ray or scanning procedures that require an injectable iodinated contrast drug and those older than 80 years.

**Comment by nelsonic** [comment]
List of side effects with probabilities: https://www.drugs.com/sfx/metformin-side-effects.html

**Comment by nelsonic** [comment]
https://www.science.org/content/article/rare-genital-defects-seen-sons-men-taking-major-diabetes-drug

Metformin, a first-line diabetes drug used for decades, may boost the risk of birth defects in the offspring of men who took it during sperm development, according to a large Danish study. Sons born to those men were more than three times as likely to have a genital birth defect as unexposed babies, according to the paper, published in the Annals of Internal Medicine today.

https://www.acpjournals.org/doi/10.7326/M21-4389 Published: **29 March 2022** 

## Conclusion:

Preconception paternal metformin treatment is associated with major birth defects, particularly genital birth defects in boys. Further research should replicate these findings and clarify the causation.

# Pass.

**Comment by nelsonic** [comment]
The study was not limited to Metformin. It was all insulin-related (diabetes) drugs. The point is that Metformin has been around and prescribed to _millions_ of people for long enough to form a longitudinal study. 
More _potent_ diabetes drugs such as those administered by injection will have an even greater effect on insulin / blood sugar levels and should be treated with extreme caution in people who are not diabetic.

**Comment by iteles** [comment]
I don't think there are any female-specific studies yet but this isn't exactly great 😬

**Comment by nelsonic** [comment]
@iteles there's a good reason there aren't any _specifically_ female studies:
a) fewer ladies have diabetes - so it might be more difficult to study.
b) ladies develop `Type 2` diabetes later in life - aged ≥45 years (i.e. after they've given birth to their children)

> "_diabetes is more prevalent in **males 2.3 %**, than in **females 1.4 %**_"
> via: Gender Differences in Living with Diabetes Mellitus: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3769156/

Anyway, it's not for me for the foreseeable future until there are further studies. 💭
